File: app/src/main/java/com/ragdroid/dahaka/activity/login/LoginContract.java
package com.ragdroid.dahaka.activity.login;

import com.ragdroid.dahaka.mvp.BasePresenter;
import com.ragdroid.dahaka.mvp.BaseView;

/**
 * Created by garimajain on 13/08/17.
 */

public interface LoginContract {

    interface View extends BaseView {

        void showHome();

        void setModel(LoginModel loginModel);
    }

    interface Presenter extends BasePresenter<View> {
        void onSubmitClicked();
    }

}

File: app/src/main/java/com/ragdroid/dahaka/activity/login/LoginContract.kt
package com.ragdroid.dahaka.activity.login

import com.ragdroid.dahaka.mvp.BasePresenter
import com.ragdroid.dahaka.mvp.BaseView

/**
 * Created by garimajain on 13/08/17.
 */

interface LoginContract {

    interface View : BaseView {

        fun showHome()

        fun setModel(loginModel: LoginModel)
    }

    interface Presenter : BasePresenter<View> {
        fun onSubmitClicked()
    }

}
